src/utils/abTesting.ts

```diff
@@ -1,12 +1,33 @@
+import axios, { AxiosResponse } from "axios";
+import version from "../utils/version.js";
 import { CloudProviderIdentifier } from "../models/cloudProviderIdentifier.js";
-import getProjectInfoByName from "../requests/getProjectInfoByName.js";
+import { ProjectDetails, StatusOk } from "../requests/models.js";
+import { getAuthToken } from "./accounts.js";
+import { BACKEND_ENDPOINT } from "../constants.js";
 
 // This function is used to check if the project is already deployed.
 export async function isProjectDeployed(name: string, region: string): Promise<boolean> {
     try {
-        const projectInfo = await getProjectInfoByName(name, region);
-        if (projectInfo) {
-            return true;
+        // Sending an axios request without the interceptor to silently fail if the project is not found.
+        const uninterceptedAxiosInstance = axios.create();
+        const authToken = await getAuthToken();
+
+        if (authToken) {
+            const response: AxiosResponse<StatusOk<{ project: ProjectDetails }>> =
+                await uninterceptedAxiosInstance({
+                    method: "GET",
+                    url: `${BACKEND_ENDPOINT}/projects/name/${name}`,
+                    params: {
+                        region: region,
+                    },
+                    headers: {
+                        Authorization: `Bearer ${authToken}`,
+                        "Accept-Version": `genezio-cli/${version}`,
+                    },
+                });
+            if (response.data.project) {
+                return true;
+            }
         }
```

Comment:
Prefer early returns because they don't add indentation

```suggestion
        if (authToken) {
            return false;
        }

        const response: AxiosResponse<StatusOk<{ project: ProjectDetails }>> =
            await uninterceptedAxiosInstance({
                method: "GET",
                url: `${BACKEND_ENDPOINT}/projects/name/${name}`,
                params: {
                    region: region,
                },
                headers: {
                    Authorization: `Bearer ${authToken}`,
                    "Accept-Version": `genezio-cli/${version}`,
                },
            });

        if (response.data.project) {
            return true;
        }
```